### ISSUE TITLE
Bump FastApi version

### ISSUE THREAD

**Issue by farmovit** [sijokun/async-fastapi-jwt-auth#9]
FastApi already has v0.105 version. Is there a reason why `async-fastapi-jwt-auth` requires v0.100 ?

**Comment by dminhhoang26** [comment]
hope `async-fastapi-jwt-auth` will not require specific fastapi version

**Comment by ilyatbn** [comment]
just tried it with the latest fastapi and its working fine (tried creating and validating jwt with cookies).. no reason not to bump it..

in the meantime for whoever wants to use it you can add the following your requirements.txt
this will just install this package without the poetry deps
```
async-fastapi-jwt-auth @ git+https://github.com/sijokun/async-fastapi-jwt-auth.git@master
PyJWT==2.8.0
httpx==0.26.0
cryptography==41.0.7
```

**Comment by sijokun** [comment]
I have released 0.6.3 with newer dependencies. Also, feel free to create PRs, i promise to merge them quickly :)
